Validation/EventGenerator/plugins/BasicHepMCValidation.cc

```diff
@@ -173,12 +184,41 @@ void BasicHepMCValidation::analyze(const edm::Event& iEvent,const edm::EventSetu
   genPtclNumber->Fill(log10(myGenEvent->particles_size()),weight);     
   genVrtxNumber->Fill(log10(myGenEvent->vertices_size()),weight);
 
-  ///Bjorken variable from PDF
+  ///PDF informations
   HepMC::PdfInfo const *pdf = myGenEvent->pdf_info();    
   if(pdf){
     bjorken = ((pdf->x1())/((pdf->x1())+(pdf->x2())));
+    logQScale = log10(pdf->scalePDF());
+    if(logQScale > 4.) logQScale = 3.995; // visualize overflow & underflow in the histograms
```

Comment:
for maintainability would not be better to have a parameter that sets both the limits of the histogram range and the safety value here?

Reply:
Okay. I'll improve that part by introducing some global parameters for upper bound & lower bound of the histograms.